### ISSUE TITLE
Add check for minishift version into integration tests

### ISSUE THREAD

**Issue by agajdosi** [minishift/minishift#1192]
Recently we had an issue in downstream with incorrect version format. To cover this we need to add scenario for minishift version, which will need to implement regexp support into code.

Syntax in Cucumber for these new regexp steps could be for example:
```
Then stdout should match "___"
And stderr should not match "___"
```

**Comment by hferentschik** [comment]
For what it's worth, we cover this in unit test as well. Of course it does to hurt to add it on an integration test level as well.

**Comment by agajdosi** [comment]
@hferentschik Not sure if this will catch something in upstream really. Main plan here is to introduce regexp support and add version check later in downstream, there we had problem with version string in RC2 last week.

**Comment by hferentschik** [comment]
Merged via pull request #1205
